apps/server/src/modules/deletion/uc/builder/deletion-request-body-props.builder.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { DeletionDomainModel } from '../../domain/types/deletion-domain-model.enum';
+import { DeletionRequestBodyPropsBuilder } from './deletion-request-body-props.builder';
+
+describe(DeletionRequestBodyPropsBuilder.name, () => {
+	afterAll(() => {
+		jest.clearAllMocks();
+	});
+
+	it('should build generic deletionRequestBodyParams with all attributes', () => {
```

Comment:
Please add proper format for describing test cases and test sets ("when... should...").

Reply:
done